bedrock/events/tests/test_data/reps.ical

```diff
@@ -22,17 +22,17 @@ BEGIN:VEVENT
 UID:https://reps.mozilla.org@2080
 CLASS:PUBLIC
 DTSTAMP:20141007T184543
-DTSTART;TZID=Africa/Johannesburg:20141008T060000
-DTEND;TZID=Africa/Johannesburg:20141009T130000
-LOCATION:Mary Fitzgerald Square\, Newtown\, Johannesburg\, Johannesburg\, 
+DTSTART;VALUE=DATE:20150719
+DTEND;VALUE=DATE:20150720
+LOCATION:Mary Fitzgerald Square\, Newtown\, Johannesburg\, Johannesburg\,
  South Africa
 SUMMARY:Tech 4 Africa
-DESCRIPTION:Tech4Africa is a place for new ideas and to encourage people t

```

Comment:
Oops. Didn't realize that my helpful editor had stripped a bunch of whitespace and made this diff so huge. Anyone care? I can revert if need be.